.travis.yml

```diff
@@ -3,4 +3,4 @@ sudo: false
 dist: trusty
 
 script:
-  - npm run lint .
+  - npm run lint . && npm test
```

Comment:
Thanks for the feedback.

That was intended. My rationale was that eslint tend to catch simple mistakes we sometimes overlook, and also forces the code format before anyone can run any tests.

In your local environment you can still do linting and testing as you wish, of course.

Was it a bad call?